docker/1.5.1/py2/Dockerfile.eia

```diff
@@ -0,0 +1,74 @@
+FROM ubuntu:16.04
+
+LABEL maintainer="Amazon AI"
+
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+ARG MMS_VERSION=1.0.5
+ARG PYTHON=python
+ARG HEALTH_CHECK_VERSION=1.3.3
+
+RUN apt-get update && \
+    apt-get -y install --no-install-recommends \
+    build-essential \
+    ca-certificates \
+    curl \
+    git \
+    libopencv-dev \
+    openjdk-8-jdk-headless \
+    python-dev \
+    python-pip \
+    vim \
+    wget \
+    zlib1g-dev \
+ && rm -rf /var/lib/apt/lists/*
+
+# See http://bugs.python.org/issue19846
+ENV LANG=C.UTF-8 \
+    PYTHONDONTWRITEBYTECODE=1 \
+    PYTHONUNBUFFERED=1 \
+    LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/usr/local/lib" \
+    PYTHONIOENCODING=UTF-8 \
+    LC_ALL=C.UTF-8
+
+WORKDIR /
+
+RUN wget https://amazonei-healthcheck.s3.amazonaws.com/v${HEALTH_CHECK_VERSION}/ei_health_check_${HEALTH_CHECK_VERSION}.tar.gz -O /opt/ei_health_check_${HEALTH_CHECK_VERSION}.tar.gz \
+ && tar -xvf /opt/ei_health_check_${HEALTH_CHECK_VERSION}.tar.gz -C /opt/ \
+ && rm -rf /opt/ei_health_check_${HEALTH_CHECK_VERSION}.tar.gz \
+ && chmod a+x /opt/ei_health_check/bin/health_check
+
+RUN pip install --no-cache-dir --upgrade \
+    pip \
+    setuptools
+
+RUN pip install --no-cache-dir \
+    https://s3.amazonaws.com/amazonei-apachemxnet/amazonei_mxnet-1.5.1-py2.py3-none-manylinux1_x86_64.whl \
+    mxnet-model-server==$MMS_VERSION \
+    keras-mxnet==2.2.4.1 \
+    # setuptools<45.0.0 because support for py2 stops with 45.0.0
+    # https://github.com/pypa/setuptools/blob/master/CHANGES.rst#v4500
+    "setuptools<45.0.0" \
+    onnx==1.4.1 \
+    # use sagemaker-inference version comaptible with MMS_VERSION=1.0.5
```

Comment:
Do we need Numpy package. if yes, we can have numpy==1.16.5.

Reply:
Looks like the 1.4.1 EI dockerfile did not have numpy. Is there a reason to add it here?